integration-service-mesh-hub.md

```diff
@@ -1,37 +1,41 @@
 # Service Mesh Hub by Solo.io
 
 The Service Mesh Hub is an example integration of the SMI Specification.
-Service Mesh Hub is a mesh agnostic dashboard to install and operate meshes. A read only version of the Service Mesh Hub can be found on [servicemeshhub.io](https://servicemeshhub.io)
+Service Mesh Hub is a mesh agnostic dashboard to install and operate meshes. A
+read only version of the Service Mesh Hub can be found on
+[servicemeshhub.io](https://servicemeshhub.io)
 
 Service Mesh Hub does:
 
 * Built-in installers for different meshes
-* Automatically discovers existing meshes, their configurations, running services / extensions and registers them to the Hub to manage
-* Install and manage meshes and mesh extensions  
-* Extensions Tab has ecosystem tools for service mesh environments. Users can also build their own extenstions
+* Automatically discovers existing meshes, their configurations, running
+ services / extensions and registers them to the Hub to manage
+* Install and manage meshes and mesh extensions
+* Extensions Tab has ecosystem tools for service mesh environments. Users can
+ also build their own extensions
 * Demos Tab has demo apps for trying out mesh use cases
 
 ## Instructions to install Service Mesh Hub
 
 Install Service Mesh Hub on any Kubernetes cluster with the command:
 
-```bash
-kubectl apply -f https://raw.githubusercontent.com/solo-io/service-mesh-hub/master/install/service-mesh-hub.yaml
+```bash kubectl apply -f
+https://raw.githubusercontent.com/solo-io/service-mesh-hub/master/install/service-mesh-hub.yaml
```

Comment:
```suggestion
```bash
kubectl apply -f
https://raw.githubusercontent.com/solo-io/service-mesh-hub/master/install/service-mesh-hub.yaml
```

Reply:
Yep, just saw that. Yay for overly helpful `vi` commands :)